### ISSUE TITLE
No module named 'rich'

### ISSUE THREAD

**Issue by jneidel** [investoreight/i8-terminal#87]
On `sudo pip install i8-terminal` and `pip install i8-terminal -U` I get:

```
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: i8-terminal in ./.local/lib/python3.10/site-packages (0.2.64)
Collecting i8-terminal
  Downloading i8-terminal-0.2.66.tar.gz (179 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 179.3/179.3 kB 1.7 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [19 lines of output]
      Traceback (most recent call last):
        File "/usr/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 353, in <module>
          main()
        File "/usr/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 335, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/usr/lib/python3.10/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py", line 118, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/tmp/pip-build-env-qllafqf8/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 338, in get_requires_for_build_wheel
          return self._get_build_requires(config_settings, requirements=['wheel'])
        File "/tmp/pip-build-env-qllafqf8/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 320, in _get_build_requires
          self.run_setup()
        File "/tmp/pip-build-env-qllafqf8/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 484, in run_setup
          super(_BuildMetaLegacyBackend,
        File "/tmp/pip-build-env-qllafqf8/overlay/lib/python3.10/site-packages/setuptools/build_meta.py", line 335, in run_setup
          exec(code, locals())
        File "<string>", line 5, in <module>
        File "/tmp/pip-install-pe1eqq9j/i8-terminal_92af71b13f9e4b34962e4db97f9779d8/i8_terminal/utils.py", line 5, in <module>
          from rich.console import Console
      ModuleNotFoundError: No module named 'rich'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

Installing rich explicitly does not change anything.

**Comment by investoreight** [comment]
Thanks @jneidel for reporting this. Which OS/platform are you using?

**Comment by jneidel** [comment]
```
$ uname -a
Linux jneidel-x240 6.2.1-arch1-1 #1 SMP PREEMPT_DYNAMIC Sun, 26 Feb 2023 03:39:23 +0000 x86_64 GNU/Linux
```

**Comment by investoreight** [comment]
@jneidel the issue should be fixed now (as of version 0.2.67)
